lib/user.rb

```diff
@@ -1,11 +1,14 @@
+require 'pry'
 require './lib/game_board'
+require 'csv'
 
 class User 
-    attr_reader :game_piece, :game_board
```

Comment:
User name will be helpful for when we add a 2 human player options, good job on that